src/components/Reports/Reports.js

```diff
@@ -27,22 +27,24 @@ const Reports = ({ dependencies: { datahubClient } }) => {
   const [state, setState] = useState({
     periodSelectedDays: periodSelectedDaysDefault,
     dateFrom: addDays(today, periodSelectedDaysDefault * -1),
-    dateTo: today,
+    dateTo: periodSelectedDaysDefault ? today : new Date(),
+    isTodaySelected: !periodSelectedDaysDefault,
```

Comment:
I would use something more general, like `dailyView`or something like that, I would think _today view_ as an special case of a _daily view_.

Reply:
changed